src/api/java/appeng/api/storage/IStorageMonitorableAccessor.java

```diff
@@ -0,0 +1,25 @@
+package appeng.api.storage;
+
+
+import javax.annotation.Nullable;
+
+import appeng.api.networking.security.BaseActionSource;
+
+
+/**
+ * Allows storage buses to request access to another ME network so it can be used as a subnetwork.
+ * This interface is used in conjunction with capabilities, so when an object of this is obtained,
+ * it already knows about which face the access was requested from.
+ * <p/>
+ * To get access to the capability for this, use @CapabilityInject with this interface as the argument
+ * to the annotation.
+ */
+public interface IStorageMonitorableAccessor
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "IStorageMonitorableAccessor" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)